### DISCUSSION TITLE
Overriding the Application.cshtml layout stops style/script bundles from loading

### DISCUSSION THREAD

**Discussion by ghost** [abpframework/abp#7761] [Q&A]
As in the title, I am trying to override the Application theme to add a sidebar to the site, as with other overrides I have created a copy of the file in Themes/Basic/Layouts. Running the app loads our unstyled welcome page with the following console errors:

ApplicationConfigurationScript:3 Uncaught ReferenceError: $ is not defined
    at ApplicationConfigurationScript:3
    at ApplicationConfigurationScript:1295
(anonymous) @ ApplicationConfigurationScript:3
(anonymous) @ ApplicationConfigurationScript:1295
ServiceProxyScript:12 Uncaught ReferenceError: abp is not defined
    at ServiceProxyScript:12
    at ServiceProxyScript:59
    at ServiceProxyScript:212

Any clues as to what I'm missing? Do I need to state elsewhere that I'm overriding the theme to load the bundles? If I manually add all the scripts/styles the whole site runs fine, so I assume its just a bundle loading issue. Has anyone else experienced this?

Edit: This is on ABP 4.2.1, to reproduce the issue copy the Application.cshtml file from the git into your own project under Pages/Themes/Basic/Layouts then run your app, if reproduced the above errors will be in the inspector console

**Comment by maliming** [comment]
* Your **ABP Framework version**.
* Steps needed to **reproduce** the problem.

**Reply by ghost** [reply]
Updated
